wiki/Tournaments/TWC/2024/en.md

```diff
@@ -149,7 +156,7 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/253
   4. [siqlo - wholeheartedly (Mew) \[haphazardly\]](https://osu.ppy.sh/beatmapsets/2152039#taiko/4534272)
   5. [xi - Garyou Tensei (Xay) \[Happy Little Streams\]](https://osu.ppy.sh/beatmapsets/2152045#taiko/4534285)
 - Hidden
-  1. [Hamu feat. Natsuki Karin - Sunao ni Naritai (Quorum) \[Hidden Answers\]](https://osu.ppy.sh/beatmapsets/2152060#taiko/4534320)
+  1. [Hamu feat. Natsuki Karin - Sunao ni Naritai (Quorum) \[Hidden Answers (osu! Edit Ver.)\]](https://osu.ppy.sh/beatmapsets/2152060#taiko/4534320)
```

Comment:
?

![image](https://github.com/ppy/osu-wiki/assets/36758269/e7de4292-8802-423f-bc10-29788e23c597)

Reply:
it's going to be updated over the following hour (I can predict the future)

Reply:
the change is supposed to be the song name (oops), fixed in c8274a4